tardis/io/tests/data/csvy_full.csvy

```diff
@@ -0,0 +1,24 @@
+---
+name: csvy_full
+description: Example csvy config file for TARDIS.
+tardis_model_config_version: v1.0
+atom_data: path/to/atom_data
+datatype:
+  fields:
+    -  name: velocity
+       unit: km/s
+       desc: velocities of shell outer bounderies.
```

Comment:
this is not required, right?

Reply:
No desc is not required.  It would have to be explicitly listed as a required property.  I did a quick test to verify.